threat_model.md

```diff
@@ -0,0 +1,190 @@
+| App secret + repeatable data access                | Tanker control        | Full access to future information    |
+| Identities access                                  | Locked device         | Targeted access to past information  |
+
+### Detailed attacks analysis
+
+**Identities + data access + Tanker control**
+
+*Attack*: The attacker gets the [User Secret] from the *identities*. They get the encrypted [Verification Key] from *Tanker*'s identity verification service. Using both of those, they are able to decrypt the [Verification Key]. Using the [Device Encryption Key Pair] contained in the [Verification Key] and the encrypted [User Encryption Key Pair] in the *ghost device*'s creation block, they get the [User Encryption Key Pair]. Using the [User Encryption Key Pair] and the key publish blocks, they can get the [Resource Encryption Key] needed to decrypt the encrypted user data.
+
+*Impact*: The attacker gains access to all past encrypted data.
+
+**Identities + repeatable data access + Tanker control**
+
+*Attack*: Same as *Identities leak + data leak + Tanker control*, but the attacker can repeat the data leak, and so gain access to future encrypted data as well.
+
+*Impact*: The attacker gains access to all past encrypted data, and to all future encrypted data.
+
+**App secret + repeatable data access + Tanker control**
+
+*Attack*: The attacker uses the [Trustchain Signature Key Pair] contained in the app secret to create new valid *devices* for every *user* in the *Trustchain*, creating an alternate version where they control every single *device*. They then perform a split view attack, showing each *user* a different view of the *Trustchain*: each *user* still sees their own *devices* correctly but every other *users*' *devices* are replaced by the attacker's *devices*. New shared data will now be shared with the attacker's controlled *device*.
+
+*Impact*: The attacker gains access to all future shared data undetected as long as they are able to maintain the split view.
+*Note*: The split view attack prevents the attacked *users* to realize they are being attacked. The attack causes some discrepancies in the way the application works, but those are hard to notice and link to an actual attack. The split view attack also requires a very dedicated attacker, having full control and knowledge of the *Tanker* server.
+*Note 2*: this attack is much more difficult to perform, and has less potential impact than *Identities leak + repeatable data leak + Tanker control*.
+
+**Identities access + locked device**
+
+*Attack*: The attacker gets the [User Secret] from the *identities*. They get the encrypted [Local Encrypted Storage] from the *device*. Using the [User Secret], they decrypt the [Local Encrypted Storage] and gain access to all past [Resource Encryption Key]s the *user* had access to. They then use the [Resource Encryption Key]s on the encrypted data stored on the *device* to access clear data.
```

Comment:
encrypted [Local Encrypted Storage] -> remove duplicate **encrypted**

Reply:
> ...on the encrypted data stored on the *device* to access clear data.

Provided there is encrypted data stored on the device. Probably the case in mobile apps, not so probable in web app.

Reply:
Alex is also mentioning that:  identity + locked device  = ability to start a Tanker session !